pkg/cmd/repo/deploy-key/list/list.go

```diff
@@ -18,6 +18,15 @@ type ListOptions struct {
 	IO         *iostreams.IOStreams
 	HTTPClient func() (*http.Client, error)
 	BaseRepo   func() (ghrepo.Interface, error)
+	Exporter   cmdutil.Exporter
+}
+
+var deployKeyFields = []string{
+	"id",
+	"key",
+	"title",
+	"created_at",
+	"read_only",
```

Comment:
We should be using camel case for field names to match what is done elsewhere. Similar comment for `secretFields`.

Reply:
Thanks @samcoe ! Updated the fields to camelCase.